### PR TITLE
feat: Get bank pv

### LINKED ISSUE #84: [Task] Gets the PV set for a bank (feature | tests | docs)
# Task

Contributors are able to complete this task and earn thenewboston coins.  Check out the labels to learn how much you can earn for contributing by completing this task.  Please make sure to be honest if you wish to contribute by saying you can't finish this and we can just un-assign you with no harm done!  There is no point in delaying tasks from being completed for miscommunication!

## Overview

Name of function | Class To add Function | Location in Account Manager | Function Description | Comments
-- | -- | -- | -- | --

getBankPV | Bank | src/renderer/dispatchers/app.ts | Gets the PV set for a bank |  
-- | -- | -- | -- | --

## Behavior
- Resolves: #73 
- user should be able to get the PV set for a bank using the SDK
- changes to existing code should be backward compatible if possible
- tests should be provided for new methods / updated for old ones
- documentation should be provided for new method / updated for old ones

```ts
/* code examples are awesome */
```

### PR DESCRIPTION
**Pull request by AbhayAysola**
Closes #84 
Account Number: 8928526805de48e4bf2ed2a9b4b839e6b2603018ecdfbf4cffdb2065e01a3ed1

### CODE REVIEW

**Comment by AbhayAysola** [comment]
The `Bank` class doesn't have any tests written yet, so I was thinking that you could make another issue and I'll write tests in a seperate PR @zinoadidi

File: src/bank.ts [hunk 1]
```diff
@@ -141,4 +141,12 @@ export class Bank extends ServerNode {
   async getValidators(options: Partial<PaginationOptions> = {}) {
     return await super.getPaginatedData<PaginatedValidatorEntry>("/validators", options);
   }
+
+  /**
+   * Gets the PrimaryValidator for the current bank.
+   */
+  async getBankPV() {
+    const { primary_validator } = await await this.getConfig();
```
Comment (zinoadidi) [inline src/bank.ts:149]: this looks like a new way to write awaits, does it work?

**Review by zinoadidi — CHANGES_REQUESTED** [review]

**Comment by zinoadidi** [comment]
test bank class task created https://github.com/thenewboston-developers/thenewboston-js/issues/90

File: src/bank.ts (hunk 1 above)
Comment (AbhayAysola) [reply src/bank.ts]: Haha I accidentally wrote await twice, my bad 😅. Not sure how I missed that

**Comment by tomijaga** [comment]
Hey, @AbhayAysola can we have the `getBankPV()` return a new `PrimaryValidator` class

**Comment by AbhayAysola** [comment]
> Hey, @AbhayAysola can we have the `getBankPV()` return a new `PrimaryValidator` class

Done

File: docs/bank.md [hunk 2]
```diff
@@ -507,3 +507,12 @@ console.log(res);
...
 
 > If you don't understand upgradeRequest and upgradeNotice, check out the [documentation](https://thenewboston.com/guide/resync-process) at thenewboston.com
+
+## Getting the Bank's Primary Validator
+
+Simply use the `Bank.getBankPV` method which returns the formatted url of the Primary Validator
+
+```ts
+const bank = new tnb.Bank("http://143.110.137.54");
+const pv = new tnb.PrimaryValidator(await bank.getBankPV());
```
Comment (tomijaga) [inline docs/bank.md:517]: Update this to reflect the new changes

**Comment by tomijaga** [comment]
@zinoadidi The PR looks good!

**Comment by zinoadidi** [comment]
Thanks @tomijaga, @AbhayAysola

**Pull request merged**